src/bench.cpp

```diff
@@ -0,0 +1,86 @@
+#include "bench.hpp"
+#include "search.h"
+#include "search.hpp"
+#include "uci.hpp"
+#include "util/types.hpp"
+#include <array>
+#include <iostream>
+#include <string>
+
+namespace Clockwork {
+namespace Bench {
```

Comment:
Minor: maybe (to reduce the number of braces):
```suggestion
namespace Clockwork::Bench {
```